client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -71,6 +71,13 @@ export function getEnhancedTasks(
 		tasks.map( ( task ) => {
 			let taskData = {};
 			switch ( task.id ) {
+				case 'setup_write':
+					taskData = {
+						title: translate( 'Personalize your site' ),
+						// eslint-disable-next-line @typescript-eslint/no-empty-function
+						actionDispatch: () => {},
```

Comment:
We can remove `actionDispatch` since users wont be interacting with this task.

Reply:
When we remove `actionDispatch` entirely, the task is hidden

Reply:
I was mistaken. I added it back in.